examples/ros_bag_recorder_adapter/adapter/globals.py

```diff
@@ -0,0 +1,3 @@
+from formant.sdk.agent.v1 import Client as FormantClient
+
+formant_client = FormantClient() 
```

Comment:
Can everything that needs formant_client be class-ified and initialize it in their __init__